test/functional/answers_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class AnswersControllerTest < ActionController::TestCase
```

Comment:
I think they should be possible to test at the controller level too -- it's still a request, and you can do xhr request testing -- see https://github.com/publiclab/spectral-workbench/blob/675d11793927fa2507eebd6c075d3304711842b7/test/functional/tags_controller_test.rb#L85